Headers/CoreGraphics/CGGeometry.h

```diff
@@ -111,13 +112,15 @@ OP_GEOM_SCOPE CGRect CGRectStandardize(CGRect rect) OP_GEOM_ATTR;
 
 /** Returns the rectangle obtained by translating rect
  * horizontally by dx and vertically by dy. */
-OP_GEOM_SCOPE CGRect CGRectOffset(CGRect rect, CGFloat dx, CGFloat dy) OP_GEOM_ATTR;
+OP_GEOM_SCOPE CGRect CGRectOffset(CGRect rect, CGFloat dx,
```

Comment:
I just checked `libs-gui` a bit and the code will go over 80 characers/line when needed to avoid awkward line breaks.
Maybe we should try 100 character columns?